src/LearnositySdk/Request/Init.php

```diff
@@ -17,6 +18,12 @@
 
 class Init
 {
+    /**
+     * We use telemetry to gather data about platforms people run our SDKs on. By default it's
+     * enabled but in special cases we need a way to disable it.
+     */
+    protected static $telemetryEnabled = true;
+
```

Comment:
@shtrom so this is added, static methods are at the bottom. Hopefully it's the last step...